### PR TITLE
upgrade moment-timezone

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: test/test_datetimeWrapper.js [hunk 1]
```diff
@@ -602,8 +602,8 @@ describe('sundial', function() {
         expect(Object.keys(tzs)).to.deep.equal(keys);
         expect(tzs.bigFour.length).to.equal(4);
         expect(tzs.unitedStates.length).to.equal(9);
-        expect(tzs.hoisted.length).to.equal(95);
-        expect(tzs.theRest.length).to.equal(441);
+        expect(tzs.hoisted.length).to.equal(98);
```
Comment (ydrisR) [inline test/test_datetimeWrapper.js:605]: What are these numbers?

**Review by ydrisR — APPROVED** [review]
Ok, but a bit more information would be welcomed ;)
For example in the PR description field: what are these changes for?

File: test/test_datetimeWrapper.js (hunk 1 above)
Comment (franck-fourel) [reply test/test_datetimeWrapper.js:605]: Don't laugh.
The world is basically split in 3:
- US timezones (big4)
- English speaking ones (hoisted)
- the rest
As we upgraded moment timezones, some new timezones are falling in the 2 last groups 😁

Comment (ydrisR) [reply test/test_datetimeWrapper.js:605]: :+1:

**Pull request merged**